### PR TITLE
fix(GlobalHeader): Catalog and Resources dropdown fast follows 

### PR DESCRIPTION
**Pull request by zippyzow**
### Overview

[DISC-731](https://codecademy.atlassian.net/browse/DISC-731?atlOrigin=eyJpIjoiNTdiNDE2MTkzMDhhNDNiNTk2N2M0NGM0ZGU0ZTdhMjciLCJwIjoiaiJ9): Cannot keydown into Inspiration section on Resources dropdown desktop
[DISC-732](https://codecademy.atlassian.net/browse/DISC-732?atlOrigin=eyJpIjoiYTk5YjY5YzNmYTMzNDY0OWJiZTc5Y2ZmY2U0MDQ0M2YiLCJwIjoiaiJ9): Change tracking target for Catalog (to expose dropdown)
[DISC-734](https://codecademy.atlassian.net/browse/DISC-734?atlOrigin=eyJpIjoiODlmYjE1MTRlYzQ3NDMxMTg0ZWEzYjJmYzM4OTU3MzQiLCJwIjoiaiJ9): Catalog and Resources dropdown menus should line up with logo for pro users *we will no longer be using the pro logo - see annotation for details

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: DISC-731. DISC-732, DISC-734
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

1. With VO on, you should now be able to keydown into Inspiration section of Resources menu
2. Catalog dropdown tracking should reflect change (see in network tab)
3. Login as a pro user or sign up as a pro user and see the regular Codecademy logo. This should also be the case for anon and free users. Resources menu should line up with Codecademy logo. 

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/31545/ 'Named link title')  | [Monolith Env](https://pr-31545-monolith.dev-eks.codecademy.com/ 'Named link title') |
| Portal       | [Portal PR](https://github.com/codecademy-engineering/portal-app/pull/2240/ 'Named link title')  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2240 'Named link title')   |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.156-alpha.56b1b8.0<br/>@codecademy/gamut-labs@32.6.1-alpha.56b1b8.0<br/>@codecademy/styleguide@58.1.11-alpha.56b1b8.0<br/></pre>

File: packages/gamut-labs/src/AppHeader/AppHeaderElements/AppHeaderResourcesDropdown/index.tsx [hunk 1]
```diff
@@ -45,9 +45,8 @@ export const AppHeaderResourcesDropdown: React.FC<AppHeaderResourceDropdownProps
   const [focusIndex, setFocusIndex] = useState(0);
 
   const items = newHeaderResourcesList.map((item) => item.data.length);
-  const itemsCount = items.reduce(
-    (prevLength, currLength) => prevLength + currLength
-  );
+  const itemsCount =
+    items.reduce((prevLength, currLength) => prevLength + currLength) + 4; // extra for hardcoded headers and descriptions
```
Comment (zippyzow) [inline packages/gamut-labs/src/AppHeader/AppHeaderElements/AppHeaderResourcesDropdown/index.tsx:49]: Fix for [DISC-731](https://codecademy.atlassian.net/browse/DISC-731?atlOrigin=eyJpIjoiNTdiNDE2MTkzMDhhNDNiNTk2N2M0NGM0ZGU0ZTdhMjciLCJwIjoiaiJ9): Cannot keydown into Inspiration section on Resources dropdown desktop

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx [hunk 2]
```diff
@@ -72,7 +63,7 @@ export const catalogDropdown = (
   icon: BookFlipPageIcon,
   id: 'catalog-dropdown',
   text: 'Catalog',
-  trackingTarget: 'topnav_catalog_dropdown',
+  trackingTarget: 'topnav_catalog',
```
Comment (zippyzow) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx:66]: Fix for [DISC-732](https://codecademy.atlassian.net/browse/DISC-732?atlOrigin=eyJpIjoiYTk5YjY5YzNmYTMzNDY0OWJiZTc5Y2ZmY2U0MDQ0M2YiLCJwIjoiaiJ9): Change tracking target for Catalog (to expose dropdown)

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62cc536808f44e3ca7672666--gamut-preview.netlify.app>https://62cc536808f44e3ca7672666--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62cc536808f44e3ca7672666>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx [hunk 3]
```diff
@@ -36,15 +36,6 @@ export const logo: AppHeaderLogoItem = {
...
 
-export const proLogo: AppHeaderLogoItem = {
-  dataTestId: 'header-pro-logo',
-  href: '/',
-  id: 'pro-logo',
-  pro: true,
-  trackingTarget: 'topnav_logo',
-  type: 'logo',
-};
-
```
Comment (zippyzow) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx:47]: Fix for [DISC-734](https://codecademy.atlassian.net/browse/DISC-734?atlOrigin=eyJpIjoiODlmYjE1MTRlYzQ3NDMxMTg0ZWEzYjJmYzM4OTU3MzQiLCJwIjoiaiJ9): Catalog and Resources dropdown menus should line up with logo for pro users 

We will no longer show the "Codecademy // Pro" logo and only show the "Codecademy" logo for all users. [Slack context](https://codecademy.slack.com/archives/C0323JNF6NL/p1657556162140039)

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderVariants.tsx [hunk 4]
```diff
@@ -247,7 +246,7 @@ export const proHeaderItems = (
   renderBookmarks?: () => ReactNode
 ): FormattedAppHeaderItems => {
   const leftItems: AppHeaderItem[] = [
-    user.hasNewSkuSubscription ? logo : proLogo,
```
Comment (zippyzow) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderVariants.tsx:250]: Relevant [portal](https://github.com/codecademy-engineering/portal-app/pull/2237) and [monolith](https://github.com/codecademy-engineering/Codecademy/pull/31535) PRs that removed this prop

**Review by borisonr — APPROVED** [review]
lgtm!

**Pull request merged**